src/modules/admin/admin.controller.js

```diff
@@ -0,0 +1,66 @@
+const Admin = require("./admin.model");
+
+const getAdmins = (req, res) => {
+  Admin.findAll()
+    .then((admins) => res.send(admins))
+    .catch((err) => console.log(err));
+};
+
+const createAdmin = (req, res) => {
+  const newAdmin = {
+    username: req.body.username,
+    password: req.body.password,
+  };
+
+  Admin.create(newAdmin)
+    .then((admin) => res.send(admin))
+    .catch((err) => console.log(err));
+};
+
+const loginAdmin = async (req, res) => {
+  const usernameInput = req.body.userNameInput;
+  const passwordInput = req.body.passwordInput;
+
+  const admin = await Admin.findOne({ where: { username: usernameInput } });
+  if (admin) {
+    if (admin.dataValues.password === passwordInput) {
+      res.send("Welcome to your dashboard");
+    } else {
+      res.send("Your password is not match");
+    }
+  } else {
+    res.send("Please enter your valid username");
+  }
+};
+
+const updateAdmin = async (req, res) => {
+  const usernameInput = req.body.usernameInput;
+  const newPassword = req.body.newPassword;
+  const confirmPassword = req.body.confirmPassword;
+
+  const findAdmin = await Admin.findOne({
+    where: { username: usernameInput },
+  });
+  if (findAdmin) {
+    if (newPassword === confirmPassword) {
```

Comment:
Vai , here you can write the logic inside a single if statement . I guess you don't need a nested if .

Reply:
bhai could you please explain how to do this